From 661031a79b1097beea8e4bdc837e538f603ab4c7 Mon Sep 17 00:00:00 2001
From: Matthew Shyu <matthew.shyu@amlogic.com>
Date: Wed, 24 Apr 2019 17:29:17 +0800
Subject: [PATCH] dts: tdmb: add samesource_srcpll to dts [1/1]

PD#SWPL-7720

Problem:
There is no audio output on HDMI on RDK platforms

Solution:
Include samesource_srcpll to g12a_s905x2_u212_drm_buildroot.dts as
g12a_s905x2_u212_buildroot.dts

Verify:
RDK/u212

Change-Id: I663aaca2e77a3a555be7d1baea28530886c50464
Signed-off-by: Matthew Shyu <matthew.shyu@amlogic.com>
---
 arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts b/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
index dd00ad385edc..cebebe0eab97 100644
--- a/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
+++ b/arch/arm64/boot/dts/amlogic/g12a_s905x2_u212_drm_buildroot.dts
@@ -897,8 +897,10 @@
 		dai-tdm-clk-sel = <1>;
 		clocks = <&clkaudio CLKID_AUDIO_MCLK_B
 				&clkc CLKID_MPLL1
-				&clkc CLKID_MPLL0>;
-		clock-names = "mclk", "clk_srcpll", "samesource_sysclk";
+				&clkc CLKID_MPLL0
+				&clkaudio CLKID_AUDIO_SPDIFOUT_CTRL>;
+		clock-names = "mclk", "clk_srcpll",
+			"samesource_srcpll", "samesource_clk";
 		pinctrl-names = "tdm_pins";
 		pinctrl-0 = <&tdmb_mclk &tdmout_b &tdmin_b>;
 		/*
-- 
2.11.0

